deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/SpecItemConfig.java

```diff
@@ -59,6 +59,12 @@ public class SpecItemConfig {
     @ConfigItem(name = "return-response")
     public Optional<Boolean> returnResponse;
 
+    /**
+     * Defines if default client header factory should be used. Default is <code>false</code>.
```

Comment:
Is this javadoc correct? Isn't `default` the default value?

Reply:
fixed